### DISCUSSION TITLE
Guide use coil3 with ktor(version 3.0.0)

### DISCUSSION THREAD

**Discussion by thanhhoai162963** [coil-kt/coil#2621] [Q&A]
I used compose mutilplatform (android + iOs). When I update coil3 (version 3.0.0-alpha06) to coil3 (version 3.0.0-rc02), I not import libary coil3, error: Could not find io.coil-kt.coil3:coil-network-ktor:3.0.0-rc02.

But when I change "io.coil-kt.coil3:coil-network-ktor" to "io.coil-kt.coil3:coil-network-ktor3" success import but ktor not run and crash app error:  java.lang.NoClassDefFoundError: Failed resolution of: Lio/ktor/client/plugins/HttpTimeout;

coil3 = "3.0.0-alpha06"
coilComposeVersion = "3.0.0-alpha06"
ktor:3.0.0

coil-compose-core = { module = "io.coil-kt.coil3:coil-compose-core", version.ref = "coil3" }
coil-network-ktor = { module = "io.coil-kt.coil3:coil-network-ktor", version.ref = "coil3" }
coil-mp = { module = "io.coil-kt.coil3:coil", version.ref = "coil3" }
coil3-coil-compose = { module = "io.coil-kt.coil3:coil-compose", version.ref = "coilComposeVersion" }

**Comment by colinrtwhite** [comment]
You need to import `ktor3` instead of `ktor`: https://coil-kt.github.io/coil/network/
